src/CodeGen_C.cpp

```diff
@@ -221,6 +222,39 @@ class HalideFreeHelper {
 } // namespace
 )INLINE_CODE";
 
+const char *const constexpr_argument_info_docs = R"INLINE_CODE(
+/**
+ * This function returns a constexpr array of information about a Halide-generated
+ * function's argument signature (e.g., number of arguments, type of each, etc).
+ * Technically this is a subset of the information provided by the existing _metadata
+ * function, except that (as a constexpr data structure), meaning one can process
```

Comment:
This sentence seems like it got a little garbled.

Reply:
Done